label-manager/createlabel/src/main/java/se/tre/freki/labelmanager/CreateLabel.java

```diff
@@ -28,27 +28,27 @@
 import javax.inject.Inject;
 
 /**
- * Command line tool to assign new IDs.
+ * Command line tool to create new Label IDs.
  */
-public final class Assign {
-  private static final Logger LOG = LoggerFactory.getLogger(Assign.class);
+public final class CreateLabel {
+  private static final Logger LOG = LoggerFactory.getLogger(CreateLabel.class);
 
   private final LabelClient labelClient;
 
   @Inject
-  Assign(final LabelClient labelClient) {
+  CreateLabel(final LabelClient labelClient) {
     this.labelClient = checkNotNull(labelClient);
   }
 
   /**
-   * Entry-point for the assign application. The assign program is normally not executed directly
-   * but rather through the main project.
+   * Entry-point for the createLabel application. The createLabel program is normally not
+   * executed directly but rather through the main project.
    *
    * @param args The command-line arguments
    */
   public static void main(final String[] args) {
     final CommandLineApplication application = CommandLineApplication.builder()
-        .command("id assign")
+        .command("id label create")
```

Comment:
This means that that [subcommand](https://github.com/hi3g/freki/pull/71/files#diff-b0080e7320f8dd7c8d2e3027dd0c67ebR26) will have to renamed as well.